bitnami/minio/templates/NOTES.txt

```diff
@@ -65,5 +65,10 @@ To access the MinIO(R) web UI:
    WARN: MinIO(R) Web UI is disabled.
 {{- end }}
 
+{{- if and (.Values.tls.enabled) (not .Values.tls.secretName) }}
+
+   NOTE: Since tls is enabled, the name of a secret (containing the certificats) has to be provided.
```

Comment:
```suggestion
   ERROR: Since TLS is enabled, the name of a secret containing the certificates has to be provided.
   {{- fail -}}
```

In this way we make the installation to fail since it will not be correct instead of just warning the issue.